package.json

```diff
@@ -65,7 +65,7 @@
     "@angular/platform-browser": "^7.2.13",
     "@angular/platform-browser-dynamic": "^7.2.13",
     "@angular/router": "^7.2.13",
-    "@hmcts/ccd-case-ui-toolkit": "^2.67.3-tab-name-replaceAll",
+    "@hmcts/ccd-case-ui-toolkit": "^2.68.9-alpha",
```

Comment:
Are you sure about releasing alpha to master?

Reply:
ii agree however until ronald comes back with the ways of semantic versioning :-)..this release for live defects is well overdue and we cant hold it back unfortunately.

Reply:
ok

Reply:
Ok